mkdocs.yml

```diff
@@ -2,7 +2,6 @@ site_name: AWS Load Balancer Controller
 repo_name: kubernetes-sigs/aws-load-balancer-controller
 repo_url: https://github.com/kubernetes-sigs/aws-load-balancer-controller
 edit_uri: edit/main/docs/
-strict: true
```

Comment:
We should remove this from the static config and pass it as an argument when building.